modules/scheduler/gke-cluster/main.tf

```diff
@@ -47,7 +85,7 @@ resource "google_container_cluster" "gke_cluster" {
 
   project         = var.project_id
   name            = local.name
-  location        = var.region
+  location        = var.is_gke_sandbox ? var.zone : var.region
```

Comment:
Can we instead name this variable as `create_multi_zonal_cluster` ?

Reply:
Introducing `cluster_availability_type` which takes the values: `{REGIONAL, ZONAL}`